packages/kbn-cli-dev-mode/src/get_server_watch_paths.ts

```diff
@@ -47,15 +47,7 @@ export function getServerWatchPaths({ pluginPaths, pluginScanDirs }: Options) {
         ...pluginScanDirs,
       ].map((path) => Path.resolve(path))
     )
-  );
-
-  for (const watchPath of watchPaths) {
-    if (!Fs.existsSync(fromRoot(watchPath))) {
-      throw new Error(
-        `A watch directory [${watchPath}] does not exist, which will cause chokidar to fail. Either make sure the directory exists or remove it as a watch source in the ClusterManger`
-      );
-    }
-  }
+  ).filter((path) => Fs.existsSync(fromRoot(path)));
```

Comment:
As we're now using the `pluginSearchPaths` from the config, we're adding folders that potential don't exist (mostly `../kibana-extra`). I had to reproduce the behavior from core which is to ignore the missing folders